aws-core/src/main/java/com/amplifyframework/auth/CognitoCredentialsProvider.kt

```diff
@@ -31,7 +32,7 @@ open class CognitoCredentialsProvider : AuthCredentialsProvider {
     /**
      * Request [Credentials] from the provider.
      */
-    override suspend fun getCredentials(): Credentials {
+    override suspend fun resolve(attributes: Attributes): Credentials {
```

Comment:
What can we do with the attributes? Do we need to support them for fetchAuthSession?

Reply:
Currently since we do not need the attributes I am passing `emptyAttributes()` into the resolve.